symengine/polys/uintpoly_flint.h

```diff
@@ -125,9 +125,9 @@ class UFlintPoly : public BaseType<Container, Poly>
         return r_iterator(this->template rcp_from_this_cast<Poly>(), -1);
     }
 
-    unsigned int size() const
+    int size() const
```

Comment:
@srajangarg @isuruf 
Seems like that. The change was made in this [commit](https://github.com/symengine/symengine/pull/1147/commits/a482c0815592008206485d27c49d32c330d0afbf) in `upolybase.h`. I've tried reverting this back, but then we'll have to make even more changes (currently, we just have to account for two `size()` functions, one in  `uintpoly_flint.h` and other in `uintpoly_piranha.h`).